app/src/main/java/com/hedvig/app/feature/embark/passages/audiorecorder/AudioRecorderFragment.kt

```diff
@@ -36,7 +39,10 @@ class AudioRecorderFragment : Fragment() {
                     startRecording = model::startRecording,
                     clock = clock,
                     stopRecording = model::stopRecording,
-                    submit = { /* TODO */ },
+                    submit = { filePath ->
```

Comment:
Should we obtain the `filePath` in this manner? Perhaps checking the `viewState` directly at this layer is more appropriate? 🤔